docs/source/Plugin/P012.rst

```diff
@@ -122,18 +122,22 @@ The following example displays a running X across the screen, switching between
       if [int#3] < [int#2]         // line < max lines
         let,3,[int#3]+1            // increment line
       else
-        let,3,1                    // reset line (screen full loop finished)
-        if [int#7] = [int#5]       // if current speed = speed1
-          let,7,[int#6]            // set to speed1
-         else
-          let,7,[int#5]            // set to speed2
-        endif
-        loopTimerSet_ms,1,[int#7]  // set new timer to current speed
+        let,3,1                    // reset line  (screen full loop finished)
+        event,changeSpeed          // raise change speed event

```

Comment:
Now I'm just nitpicking... 
I prefer "asyncevent" as it will not start a new rules parsing, which "event" does.